notice_and_comment/static/regulations/css/less/fonts.less

```diff
@@ -0,0 +1,29 @@
+/*
+ Fonts
+ ==========================================================================
+ Place all @font-face rules in fonts.less
+ */
+
+@font-face {
```

Comment:
If you mean having the font face declarations in the same file as the headers, they are in separate files because that's how they are being over written on -site. It does all compile to the same thing at the end, though.